platform/test/spec/testErrorHandlerSpec.js

```diff
@@ -1,27 +1,20 @@
-/*global describe, it, beforeEach, expect --  functions provided by Jasmine */
+/*global describe, it, beforeEach, expect, spyOn --  functions provided by Jasmine */
 /*global jasmine --  object provided by Jasmine */
 
 import { EducationPlatformError } from "../../src/EducationPlatformError.js";
 import {ErrorHandler} from "../../src/ErrorHandler.js"
+import { PlaygroundUtility } from "../../src/PlaygroundUtility.js";
 
 describe("ErrorHandler", () => {
 
     // Setup
     let notifierSpy;
 
     beforeEach(()=>{
-        notifierSpy = jasmine.createSpy("notifier");
+        notifierSpy = spyOn(PlaygroundUtility,"errorNotification");
```

Comment:
This is still being passed in as a parameter to the `ErrorHandler` constructor calls below. Doesn't that need fixing, too?

Reply:
Thanks yes, fixed in 27027ebcef16eba9977cddc6e63726d01cdae3a0